src/utils/ImportPathResolver.ts

```diff
@@ -32,15 +32,15 @@ import { resolve, join, dirname } from 'path';
 export type StringReplacer = (importStatement: string) => string;
 
 const anyQuote = '["\']';
-const notQuote = '[^"\']';
-const importString = `(?:${anyQuote}${notQuote}+${anyQuote})`;
+const pathStringContent = `[^"'\r\n]+`;
```

Comment:
Previously was using `noteQuote` to match characters between two quotes. But this would match anything, even if it couldn't be part of an import path. Renamed based on actual use and included newline chars in the restricted set. This prevents matches to things that definitely aren't valid import paths due to being split across lines or having internal quotes, without assuming any other restrictions on valid path characters.